### DISCUSSION TITLE
Git sync to subfolder

### DISCUSSION THREAD

**Discussion by Lavhe** [requarks/wiki#7315] [Help / Questions]
Hi

I have a bitbucket repo that looks like

```
- web
- api
- docs
   - wiki
```

And I would like wiki to just sync to the `docs/wiki` folder, I tried by all means, but it always syncs to the root of the project, is this possible?

Basically being able to specify the `root`

**Comment by dawidp** [comment]
One solution can be to make new repo for wiki and add it, to the main project, as a [submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules) in `docs/wiki` .
